### ISSUE TITLE
Finetuning the checkpoints

### ISSUE THREAD

**Issue by onirhakin** [ivonajdenkoska/tulip#3]
Hi, once the checkpoint is released, would it be possible for me to finetune it simply by performing the second step of the training?
So this one: "For the second stage of the training, we perfrom fine-tuning of the student model by optimizing the CLIP loss with the new positional encodings for a single epoch."

Let's say i want the model to be accostumed to longer prompts and maybe formatted in a specific way, like pseudo-xml, can  it work in that way?
Would it work even with a dataset of 50k images instead of 1M if i finetune on your checkpoint?

**Comment by ivonajdenkoska** [comment]
Hi @onirhakin. Thanks for your interest in our work.

Yes, you can continue fine-tuning the model by performing the second step of the training. 

To train for longer prompts, the variable student-context-length should be adjusted accordingly in order to instantiate the student model properly.

Regarding the last question, it would be possible to train with a smaller dataset, however fine-tuning with fewer images could lead to overfitting, especially if the dataset lacks sufficient variation.

**Comment by onirhakin** [comment]
Ok, thanks!
